internal/clients/diffreviewer/github/github_service.go

```diff
@@ -116,18 +100,26 @@ type CheckRequest struct {
 	Name string `json:"name,omitempty"`
 }
 
+type RepoParams struct {
+	GitURL  string
+	BaseRev string
```

Comment:
nit - should this be `BaseRef`?

Reply:
The reason I called it `BaseRev` is because depending on which action triggered the action pr or push the value will be retrieved from somewhere different. If the action is a pr then the value is `BaseRef`, if the action is push then the value is `Before` which is the hash of the last time the workflow was run on the branch. So used the name `BaseRev` to be more general.